src/xkit.cs

```diff
@@ -2167,4 +2168,476 @@ partial interface NSTextAttachment : NSTextAttachmentContainer, NSSecureCoding
 		[Export ("textAttachmentWithImage:")]
 		NSTextAttachment Create (Image image);
 	}
+
+	[NoWatch]
+	[MacCatalyst (13,0)]
+	[iOS (7,0)]
+	[BaseType (typeof (NSMutableAttributedString), Delegates=new string [] { "Delegate" }, Events=new Type [] { typeof (NSTextStorageDelegate)})]
+	partial interface NSTextStorage : NSSecureCoding {
+#if MONOMAC && !XAMCORE_4_0
+		[Export ("initWithString:")]
+		IntPtr Constructor (string str);
+#endif
+
+		[Export ("layoutManagers")]
+#if MONOMAC || XAMCORE_4_0
+		NSLayoutManager [] LayoutManagers { get; }
+#else
+		NSObject [] LayoutManagers { get; }
```

Comment:
Why not do the using NSLayoutManager  = System.NSObject[] in a #if at the start of the file?

Reply:
We could do that, but the "using ..." pattern is usually used for types that don't exist on a particular platform to make the rest of the code compile, while in this case it's a binding mistake/improvement, which is only a problem in this particular location, so you'd end up with something like this at the top of the file:

```csharp
#if MONOMAC || XAMCORE_4_0
using LayoutManagersReturnTypeForNSTextStorage = Foundation.NSLayoutManager[];
#else
using LayoutManagersReturnTypeForNSTextStorage = Foundation.NSObject[];
```

and I don't think that looks much better.